test/jdk/tools/jpackage/apps/ThirdPartyAppLauncher.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.io.IOException;
+import java.util.logging.Logger;
+
+public class ThirdPartyAppLauncher {
+    private static final Logger logger = Logger
+            .getLogger(ThirdPartyAppLauncher.class.getName());
+
+    public static void main(String[] args) throws IOException {
+        ProcessBuilder processBuilder = new ProcessBuilder("regedit");
+        Process process = processBuilder.start();
+        logger.info("RegEdit id=" + process.pid());
```

Comment:
Logging is excessive and not applicable here as it can be redirected. The test shall print the PID to stdout.

Reply:
Sure, will print the PID using sysout